docs/source/user/aerodyn/input.rst

```diff
@@ -754,3 +807,90 @@ See :numref:`ad_blade_geom`. Twist is shown in :numref:`ad_blade_local_cs` of :n
 
    AeroDyn Blade Geometry – Left: Side View; Right: Front View (Looking Downwind)
 
+
+
+.. _TF_tf_input-file:
+
+Tail fin input file
+-------------------
+
+An example of tail fin input file is given below:
+
+.. code::
+    
+    ------- TAIL FIN AERODYNAMICS INPUT FILE--------------------------------------------
+    Comment
+    ======  General inputs =============================================================
+    1         TFinMod     - Tail fin aerodynamics model {0: none, 1: polar-based, 2: USB-based} (switch)
+    1.0       TFinArea    - Tail fin planform area (m^2) [used only when TFinMod=1]
```

Comment:
It looks like the model also needs TFinChord for the tail fin pitching moment calculation.  Presumably this must be another input in the tail fin aerodynamics input file.

Reply:
That's correct, it was in the code but omitted in the documentation.